### ISSUE TITLE
forwarded logits are different on different devices

### ISSUE THREAD

**Issue by pfldy2850** [vllm-project/vllm#911]
I have A100 40GB and A100 80GB devices.
In each, I'm testing polyglot based model inference using vLLM.

During testing, I noticed that the logits forwarded on the two devices were different.

In A100 40GB,
```
>> logits
tensor([[  9.8906,  10.0391,   7.9766,  ...,  -7.6797,  -6.5195, -11.6953]],
       device='cuda:0', dtype=torch.float16)

>> torch.argmax(logits, dim=1)
tensor([202], device='cuda:0'
```

In A100 80GB,
```
>> logits
tensor([[-0.9316, -0.8354,  0.9756,  ...,  0.5137, -3.9805, -4.0625]],
       device='cuda:0', dtype=torch.float16)

>> torch.argmax(logits, dim=1)
tensor([2190], device='cuda:0')
```

Generatedd on the A100 40GB was a bit more unnatural.

Is there a possibility that the forwarded logits may vary for each device? 
Or, if there is any section you are suspicious of, please let me know and I will check it.

**Comment by pfldy2850** [comment]
I made the mistake of missing one of my model checkpoint files on my A100 40GB device.
After correcting the mistake, I tested it and the same output was obtained normally.
Issue resolved and closed.

**Comment by zdchu** [comment]
Can I ask how did you get the logits? Many thanks
